docs/contributor-guide.md

```diff
@@ -33,4 +33,10 @@ provided by the bot. You will only need to do this once across all repos using o
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
\ No newline at end of file
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Getting Started
+
+You�ll need Visual Studio 2019 installed to do work in a GUI, or at least the MSBuild and SDK toolchains to build from the command line.  From Visual Studio, open the ProjectReunion.sln and hit build!
```

Comment:
Add a link to the VS2019 installer landing page, like https://visualstudio.microsoft.com/downloads/.  Also clarify that VSCode works great too.

Reply:
Download link - good idea. Done.

VSCode - can you build from VSCode or just edit? I haven't used VSCode to compile before. Is there any particular setup needed in the project or VSCode command(s) to use it as a dev environment? Any extensions required or recommended? Better left for another more familiar with VSCode to make a future PR about it?